jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/CompressedContentFormat.java

```diff
@@ -15,20 +15,10 @@
 
 import java.util.Objects;
 
-import org.eclipse.jetty.util.QuotedStringTokenizer;
 import org.eclipse.jetty.util.StringUtil;
 
 public class CompressedContentFormat
 {
-    /**
-     * The separator within an etag used to indicate a compressed variant. By default the separator is "--"
-     * So etag for compressed resource that normally has an etag of <code>W/"28c772d6"</code>
-     * is <code>W/"28c772d6--gzip"</code>.  The separator may be changed by the
-     * "org.eclipse.jetty.http.CompressedContentFormat.ETAG_SEPARATOR" System property. If changed, it should be changed to a string
-     * that will not be found in a normal etag or at least is very unlikely to be a substring of a normal etag.
-     */
-    public static final String ETAG_SEPARATOR = System.getProperty(CompressedContentFormat.class.getName() + ".ETAG_SEPARATOR", "--");
-
     public static final CompressedContentFormat GZIP = new CompressedContentFormat("gzip", ".gz");
     public static final CompressedContentFormat BR = new CompressedContentFormat("br", ".br");
     public static final CompressedContentFormat[] NONE = new CompressedContentFormat[0];
```

Comment:
Why there is not one for `zip`? Tests are continuously creating the `zip` format when there could be a constant here.

Reply:
zip is an oddball one.
It's not a supported standard for Accept-Encoding / Content-Encoding

Reply:
Fair enough.